x-pack/plugins/osquery/cypress/tasks/navigation.ts

```diff
@@ -21,28 +21,6 @@ export const navigateTo = (page: string, opts?: Partial<Cypress.VisitOptions>) =
 
   // There's a security warning toast that seemingly makes ui elements in the bottom right unavailable, so we close it
   closeToastIfVisible();
-  waitForReact();
-};
-
-// We're moving away from using react-cypress-selector, I'll be adjusting this on file by file approach
```

Comment:
![giphy](https://github.com/elastic/kibana/assets/39014407/3c5cb622-b2f4-4f36-8443-16b47454b0ba)

because the plan has come together